### PR TITLE
Implement even faster version

### PR DESCRIPTION
**Pull request by PaulKovalov**
This is a slightly faster version of optional attendees, however I don't really like it because it is bulky and doesn't give that much performance improvement over TreeSet, which is here https://github.com/PaulKovalov/portfolio/pull/26. Just making this PR to demonstrate another possible solution.

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/MeetingRequestTest.java [hunk 1]
```diff
@@ -44,4 +44,4 @@ public void CantAddOptionalAttendeeWhoIsAlsoMandatory() {
     int expected = 0;
     Assert.assertEquals(expected, actual);
   }
-}
```
Comment (harry-7) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/MeetingRequestTest.java:47]: Ideally this should be in the previous PR?

**Comment by harry-7** [comment]
> This is a slightly faster version of optional attendees, however I don't really like it because it is bulky and doesn't give that much performance improvement over TreeSet, which is here #26. Just making this PR to demonstrate another possible solution.

I think if you look at bigger scale, this would be 10X faster atleast. which is significant improvement IMHO.

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/MeetingRequestTest.java (hunk 1 above)
Comment (PaulKovalov) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/MeetingRequestTest.java:47]: you mean whole this PR?

**Comment by PaulKovalov** [comment]
> > This is a slightly faster version of optional attendees, however I don't really like it because it is bulky and doesn't give that much performance improvement over TreeSet, which is here #26. Just making this PR to demonstrate another possible solution.
> 
> I think if you look at bigger scale, this would be 10X faster atleast. which is significant improvement IMHO.

okay, then I will use this version

**Pull request merged**